src/tests/PaginationArrow.spec.js

```diff
@@ -34,31 +32,10 @@ describe('The Pagination Arrow component', function () {
       onTrigger: clickTrigger,
     });
     expect(this.renderedComponent).toEqual(
-      <div className='my-class' onClick={clickTrigger}>
-        <div className='PaginationArrowIcon' />
+      <div bemBlock='DateRangePicker' className='DateRangePicker__PaginationArrow DateRangePicker__PaginationArrow--next' onClick={clickTrigger} >
```

Comment:
There is a small issue with the bem functionality. If we don't specify the attribute, we end up with class names such as 'null__PaginationArrowIcon'. I thought this would be a bit cleaner